### ISSUE TITLE
Update OWNERs of central dashboard

### ISSUE THREAD

**Issue by jlewi** [kubeflow/kubeflow#5151]
/kind process

The OWNERs file of the centraldashboard was last updated over a year ago.
https://github.com/kubeflow/kubeflow/blob/master/components/centraldashboard/OWNERS

It looks like at least @prodonjs has indicated [here](https://github.com/kubeflow/kubeflow/issues/3576#issuecomment-660055405) that he is no longer actively working on this.

I think @avdaredevil is also primarily focused on other projects but ocassionally pitches in.

@swiftdiaries are you still actively working on centraldashboard or willing to?

It looks like we may need to find an appropriate set of owners in order to continue to drive this forward.

**Comment by jlewi** [comment]
@swiftdiaries Are you still an active OWNER of centraldashboard if so can you can take this on?

/assign @swiftdiaries

**Comment by swiftdiaries** [comment]
I can review code for the time being but I'm not familiar enough with the current UI codebase to be an approver. We should probably advertise through the community for UI OWNERS

**Comment by SachinVarghese** [comment]
Hey, @jlewi @swiftdiaries I have been spending some time on the central dashboard codebase recently working on some PRs and have become quite familiar with many parts of the project now. 
Previously, I have been involved in contributions to other parts of the project mainly kfserving, seldon-core and oidc-authservice. And I will be happy to spend some time on building and maintaining the central dashboard project as an owner. Let me know your thoughts.

**Comment by swiftdiaries** [comment]
That's really cool, thank you. Could you open a PR updating the OWNERS file to move me (swiftdiaries) to approvers and adding yourself to the reviewers?

**Comment by SachinVarghese** [comment]
Hey, @swiftdiaries Thanks for the response. I have created a PR https://github.com/kubeflow/kubeflow/pull/5153 as you mentioned and also noticed that you are already in the approvers' list. 
As a next step, it would be great to sync up some time for me to understand other things related to the project like the code review format, pr builds, test pipelines, docs, and other relevant instructions. Let me know if there are any weekly meetings for this working group. I'm also available on kubeflow slack for any related communication. Cheers.

**Comment by StefanoFioravanzo** [comment]
Hello everyone. I am also interested in stepping up to helping out managing Centraldashboard. @jlewi You may remember that I presented a revamped Centraldashboard during the community meeting a couple of weeks ago. I have been doing some work that I will start to push upstream very soon. I'd love to become part of the reviewers as well! 

@SachinVarghese we could share the load and learn things together. Maybe I could join you in #5153 ; and similarly I would need to be added as a member of the Org as well.

**Comment by swiftdiaries** [comment]
That's amazing! Thank you so much for stepping up y'all 🙌🏾
@StefanoFioravanzo could you please open a separate PR with the change to the OWNERS file if that's ok?

**Comment by jlewi** [comment]
@SachinVarghese and @StefanoFioravanzo sounds great. My expectation is that the OWNERs of central dashboard own it E2E which includes

* Submitting and reviewing code
* Monitoring/Responding to GitHub issues related to central dashboard
* Driving feature development
* Driving and coordinating release of central dashboard
* Driving testing for centraldashboard

Are you willing to assume some or all of those responsibilities? If so please open up PRs to add yourself to the GitHub org?
https://www.kubeflow.org/docs/about/contributing/#joining-the-kubeflow-github-org
and then subsequent PRs to update the OWNERs files?

> As a next step, it would be great to sync up some time for me to understand other things related to the project like the code review format, PR builds & test pipelines, docs, and other relevant instructions. Let me know if there are any weekly meetings for this working group. I'm also available on kubeflow slack for any related communication. Cheers.

Happy to! My suggestion would be to do it in of the [regular community meetings](https://www.kubeflow.org/docs/about/community/#community-meetings) which happen on Tuesdays. 

This [thread](https://groups.google.com/forum/#!searchin/kubeflow-discuss/contributing$20to$20kubeflow%7Csort:date/kubeflow-discuss/pGflW6m9Xrg/nBjyV8QGAgAJ) has a list of links to help understand our processes.

**Comment by swiftdiaries** [comment]
There's no separate UI WG at the moment. Since the group is small and scope of the group is limited I think it's fine as it is. We can squeeze in time as part of the community meetings. If appropriate space and time is not allotted as part of the community meeting, we can always revisit this later. 

Regarding contributor / dev workflow, I believe @animeshsingh created a Kubeflow Dojo with the express intent to onboard new contributors. Could you PTAL?

**Comment by SachinVarghese** [comment]
Thanks, @jlewi, and @swiftdiaries for all the relevant links, and also I am aware of the kubeflow Dojo session videos and slides.
I have recently started to attend the community meetings on Tuesdays and @StefanoFioravanzo it would be great to work together and hope to contribute more now. I have created a PR adding myself as an org member https://github.com/kubeflow/internal-acls/pull/292 as instructed.
